packages/backend/src/assets/ai.json

```diff
@@ -12,6 +12,8 @@
       "recommended": [
+        "hf.bartowski.granite-3.1-8b-instruct-GGUF",
+        "hf.lmstudio-community.granite-3.0-8b-instruct-GGUF",
         "hf.instructlab.granite-7b-lab-GGUF",
         "hf.instructlab.merlinite-7b-lab-GGUF",
         "hf.lmstudio-community.granite-3.0-8b-instruct-GGUF"
```

Comment:
I think you should remove this entry, as it has been added at the beginning

Reply:
Fixed